### PR TITLE
Follow skip forward/back length on ExoPlayer

### LINKED ISSUE #84: Skip forward and back doesn't follow settings in exoplayer
Pressing the right side of the screen doesn't skip the set 30s but only 5s

**To Reproduce**
1. Set forward and back skip to 30 seconds in settings
2. Start video
3. Skip forward and backward by double tapping the sides of the screen

**System (please complete the following information):**
 - App version: 2.0.0-rc.2
 - Android version: Android 10
 - Device manufacturer and model: OnePlus 6t
 - Jellyfin server version: 10.6.3
 - The type of video player implementation use: integrated player (Exoplayer)

### PR DESCRIPTION
**Pull request by CarlosOlivo**
Closes #84

### CODE REVIEW

**Review by Maxr1998 — APPROVED** [review]

**Comment by Maxr1998** [comment]
Add an extra model class to store the display preferences, that way the number only has to be parsed once. Rest works well.

File: app/src/main/java/org/jellyfin/mobile/controller/ApiController.kt [hunk 1]
```diff
@@ -63,11 +72,35 @@ class ApiController(
...
+                    displayPreferencesId = "usersettings",
+                    client = "emby"
+                )
+
+                val customPrefs = displayPreferences.customPrefs
+
+                DisplayPreferences(
+                    skipBackLength = customPrefs?.get("skipBackLength")?.toLongOrNull() ?: DEFAULT_SEEK_TIME_MS,
+                    skipForwardLength = customPrefs?.get("skipForwardLength")?.toLongOrNull() ?: DEFAULT_SEEK_TIME_MS
+                )
```
Comment (nielsvanvelzen) [inline app/src/main/java/org/jellyfin/mobile/controller/ApiController.kt:94]: Should use constants for these

**Review by nielsvanvelzen — CHANGES_REQUESTED** [review]
Instead of adding a fragile HTTP interceptor it might be easier to just GET the display preferences when the player starts. That should be more reliable.

**Comment by Maxr1998** [comment]
True that, I'll do that.

**Review by nielsvanvelzen — APPROVED** [review]

**Review by Maxr1998 — APPROVED** [review]

**Pull request merged**